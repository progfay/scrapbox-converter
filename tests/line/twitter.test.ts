/* global describe it expect */

import { parse } from '@progfay/scrapbox-parser'
import convert from '../../src'
import pretty from 'pretty'

describe('twitter', () => {
  it('Simple twitter', () => {
    const input = `Twitter
[https://twitter.com/progfay/status/1190909733885972481]`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })

  it('Twitter link with statuses', () => {
    const input = `Twitter
[https://twitter.com/progfay/statuses/1190909733885972481]`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })
})
