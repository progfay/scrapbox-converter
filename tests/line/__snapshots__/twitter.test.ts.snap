// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`twitter Simple twitter 1`] = `
"<article>
  <h1 class=\\"title\\">Twitter</h1>
  <section>
    <p>
      <amp-twitter width=\\"375\\" height=\\"472\\" layout=\\"responsive\\" data-tweetid=\\"1190909733885972481\\"></amp-twitter>
    </p>
  </section>
</article>"
`;

exports[`twitter Twitter link with statuses 1`] = `
"<article>
  <h1 class=\\"title\\">Twitter</h1>
  <section>
    <p>
      <amp-twitter width=\\"375\\" height=\\"472\\" layout=\\"responsive\\" data-tweetid=\\"1190909733885972481\\"></amp-twitter>
    </p>
  </section>
</article>"
`;
